frontend/src/components/CustomAlertBar.tsx

```diff
@@ -0,0 +1,49 @@
+import { useAppSelector } from '@/custom-hooks/StateHooks';
```

Comment:
Did you create this alert bar only for error messages? If so, please rename the file to CustomErrorAlert.

Reply:
It is both for error message and success message

Reply:
Why not name it `Snackbar`?

Reply:
Changed the name to SnackBar

Reply:
We already have a library for Snackbar, but we can add a prefix to avoid confusion.